modules/instance/versions.tf

```diff
@@ -15,5 +15,10 @@
  */
 
 terraform {
-  required_version = "~> 0.12.6"
+  required_version = ">= 0.13"
+
+  provider_meta "google" {
```

Comment:
Lets add the beta too since there is a beta resource https://github.com/terraform-google-modules/terraform-google-data-fusion/blob/b9319240e2bb39713aa112c8104f4ab4ca34c1f6/modules/instance/main.tf#L18

Reply:
👍